pkgs/applications/video/natron/default.nix

```diff
@@ -88,9 +95,10 @@ stdenv.mkDerivation {
   inherit version;
   name = "natron-${version}";
```

Comment:
```suggestion
  pname = "natron";
```

Reply:
Shall I also change `name = "openfx-${pluginName}-${version}";` (line 25) to `pname = "openfx-${pluginName}";`?

Reply:
Yes, please if it uses mkDerivation.